LocalPayment/src/main/java/com/braintreepayments/api/localpayment/LocalPaymentRequest.kt

```diff
@@ -0,0 +1,171 @@
+package com.braintreepayments.api.localpayment
+
+import com.braintreepayments.api.core.PostalAddress
+import org.json.JSONException
+import org.json.JSONObject
+
+/**
+ * Builder used to construct an local payment request.
+ */
+class LocalPaymentRequest
+/**
+ * @param hasUserLocationConsent is an optional parameter that informs the SDK
+ * if your application has obtained consent from the user to collect location data in compliance with
+ * [Google Play Developer Program policies](https://support.google.com/googleplay/android-developer/answer/10144311#personal-sensitive)
+ * This flag enables PayPal to collect necessary information required for Fraud Detection and Risk Management.
+ *
+ * @see [User Data policies for the Google Play Developer Program ](https://support.google.com/googleplay/android-developer/answer/10144311.personal-sensitive)
+ *
+ * @see [Examples of prominent in-app disclosures](https://support.google.com/googleplay/android-developer/answer/9799150?hl=en.Prominent%20in-app%20disclosure)
+ */(private val hasUserLocationConsent: Boolean) {
+    /**
+     * @param address Optional - The address of the customer. An error will occur if this address is not valid.
+     */
+    @JvmField
+    var address: PostalAddress? = null
```

Comment:
Could we move these fields into the primary constructor? This way we'd be able to make it a data class.